node/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -148,6 +156,9 @@ data WalletStorage = WalletStorage
     , _wsTxHistory       :: !(HashMap (CId Wal) (HashMap CTxId CTxMeta))
     , _wsHistoryCache    :: !(HashMap (CId Wal) (Map TxId TxHistoryEntry))
     , _wsUtxo            :: !Utxo
+    -- @_wsBalances@ depends on @_wsUtxo@,
+    -- it's forbidden to update @_wsBalances@ without @_wsUtxo@
```

Comment:
In order to reduce the chances of someone accidentally modifying the `wsBalances` without reading this comment, can we exclude the field accessor and lens from the export list?